.github/workflows/tests.yml

```diff
@@ -146,10 +162,17 @@ jobs:
       arch: ${{ matrix.arch }}
       flavor: ${{ matrix.flavor }}
   test_platform_flavors:
-    needs: intersect_matrices
+    needs: [ intersect_matrices, test_flavors_chrooted ]
```

Comment:
I'd suggest also making the platform tests dependent on the QEMU tests. That way the QEMU tests can already catch any issues preventing machines from booting at all => we don't waste resources on starting cloud VMs that will never even boot.